.storybook/stories/GenericPromoBanner.stories.js

```diff
@@ -70,7 +26,7 @@ export const Default = () => ({
 	props: {
 		iconKey: {
 			type: String,
-			default: select('Icon Key', iconKeySelect, 'present')
+			default: 'present'
```

Comment:
Sorry @eddieferrer, since these icons are imported directly now I couldn't get them to swap out dynamically. I added a story to loop through all the available contentful icons instead.

Reply:
Unfortunate, but should be fine. 👍